templates/dotnet/publish-nuget-packages.yml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  - name: nugetProjectPath
+    type: string
+
+  - name: vstsFeed
+    type: string
+    default: 'Video Hearings/vh-packages'
+
+steps:
+- task: gitversion/setup@0
+  displayName: Install Git Version
+  inputs:
+    versionSpec: '5.6.0'
+    
+- task: gitversion/execute@0
+  displayName: Run Git Version
+            
+- task: DotNetCoreCLI@2
+  displayName: dotnet restore
+  inputs:
+    command: restore
+    projects: ${{ parameters.nugetProjectPath }}/*.csproj
+    vstsFeed: ${{ parameters.vstsFeed }}
+    verbosityLevel: Minimal
+
+- task: DotNetCoreCLI@2
+  displayName: Create Build NuGet Project
+  inputs:
+    arguments: '--configuration release /p:Version="$(MajorMinorPatch)" /p:PackageOutputPath="$(Build.ArtifactStagingDirectory)" /p:VersionSuffix="-alpha"'
```

Comment:
I think the suffix is what we will need to use later for the other project, so might be worth making this a parameter

Reply:
I don't think the suffix has any effect on what is published to Azure DevOps, I cannot see the reference Alpha anywhere with all the new packages I have deployed yesterday, it looks like it is just Version= which is used.

I have made the Version an optional parameter for now, defaulting to use GitVersion if one is not specified.